{u'description': [u'Gentoo cross-compilation overlay for embedded devices'],
 u'feeds': [u'https://cgit.gentoo.org/proj/embedded-cross.git/atom/'],
 u'homepage': u'https://cgit.gentoo.org/proj/embedded-cross.git/',
 u'irc': None,
 u'name': u'embedded-cross',
 u'official': True,
 u'owner_email': u'armin76@gentoo.org',
 u'owner_name': u'Ra\xfal Porcel',
 u'priority': 50,
 u'quality': u'experimental',
 u'sources': [(u'git://anongit.gentoo.org/proj/embedded-cross.git',
               u'Git',
               u''),
              (u'https://anongit.gentoo.org/git/proj/embedded-cross.git',
               u'Git',
               u''),
              (u'git+ssh://git@git.gentoo.org/proj/embedded-cross.git',
               u'Git',
               u'')],
 u'status': u'official',
 u'supported': True}
$ pmaint sync /home/mgorny/repos/embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/clickbeetle_tools', named 'clickbeetle_tools', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/armagetron', named 'armagetron', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/arcon' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/arcon', named '<unlabeled repository /home/mgorny/repos/arcon>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/bicatali', named 'bicatali', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/dauleet' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/dauleet', named '<unlabeled repository /home/mgorny/repos/dauleet>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/docd', named 'docd', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/alice', named 'alice', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/Menelkir', named 'Menelkir', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at location '/home/mgorny/repos/arcon' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/arcon', named '<unlabeled repository /home/mgorny/repos/arcon>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/efika' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/efika', named '<unlabeled repository /home/mgorny/repos/efika>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/dauleet' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/dauleet', named '<unlabeled repository /home/mgorny/repos/dauleet>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/armagetron', named 'armagetron', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/Menelkir', named 'Menelkir', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/clickbeetle_tools', named 'clickbeetle_tools', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/docd', named 'docd', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/edwtjo-ebuilds' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/edwtjo-ebuilds', named '<unlabeled repository /home/mgorny/repos/edwtjo-ebuilds>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/alice', named 'alice', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/drizzt-overlay' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/drizzt-overlay', named '<unlabeled repository /home/mgorny/repos/drizzt-overlay>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/bicatali', named 'bicatali', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
Already up-to-date.
*** syncing '/home/mgorny/repos/embedded-cross'...
*** synced '/home/mgorny/repos/embedded-cross'
 * Sync succeeded
$ pmaint regen -t 2 embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'

!!! ERROR: app-misc/mdbus-9999 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: x11-terms/vala-terminal-1.1.1 failed.

!!! ERROR: x11-terms/vala-terminal-1.2 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:

!!! ERROR: dev-libs/libnl-2.0_pre9999 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '208' 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '218' 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '165' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
 * Use PYTHON_DEPEND variable instead of NEED_PYTHON variable.

!!! ERROR: dev-python/gst-python-0.10.16 failed.
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '204' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-libs/libnl/libnl-2.0_pre9999.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-libs/libnl/libnl-2.0_pre9999.ebuild'
 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-9999.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-libs/libnl/libnl-2.0_pre9999.ebuild'

!!! ERROR: media-plugins/gst-plugins-ivorbis-0.10.9 failed.
 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!       libnl-2.0_pre9999.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: dev-python/pyrtc-9999 failed.
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '215' 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:

!!! ERROR: dev-libs/libgee-0.5.0.1 failed.
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'

!!! ERROR: dev-python/pyrtc-20090418 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:

!!! ERROR: x11-drivers/xf86-video-glamo-9999 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:

!!! ERROR: dev-python/python-elementary-20090614 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!       vala-terminal-1.2.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                  ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                  ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!     vala-terminal-1.1.1.ebuild, line   5:  Called inherit 'git' 'autotools'

!!! ERROR: sys-devel/gcc-4.4.3 failed.
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                  ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!  inherit for git failed

!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/gst-python/gst-python-0.10.16.ebuild'

!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!! ERROR: dev-util/dialog-1.1.20080819 failed.
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                  ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                  ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '251' 'depend'

!!! ERROR: dev-python/python-ecore-20090729 failed.
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!              mdbus-9999.ebuild, line   5:  Called inherit 'git'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                  ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '261' 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:

!!! ERROR: x11-drivers/xf86-video-glamo-20100626 failed.
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/gst-python/gst-python-0.10.16.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '189' 'depend'

!!! ERROR: sys-devel/gcc-4.4.2 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '228' 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '166' 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/gst-python/gst-python-0.10.16.ebuild'
 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/python-e_dbus/python-e_dbus-20090614.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '169' 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                  ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!  inherit for git failed

!!!                  ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '189' 'depend'
!!!                 ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '214' 'depend'

!!! ERROR: dev-python/python-edje-20090614 failed.
!!!                          ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-elementary/python-elementary-20090614.ebuild'
!!!      gst-python-0.10.16.ebuild, line   7:  Called inherit 'autotools' 'eutils' 'multilib' 'python'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                 ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                          ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-elementary/python-elementary-20090614.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.3.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'python'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '226' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                          ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/media-plugins/gst-plugins-ivorbis/gst-plugins-ivorbis-0.10.9.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-libs/libgee/libgee-0.5.0.1.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '166' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.3.ebuild'
!!!                 ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '248' 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-9999.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-util/dialog/dialog-1.1.20080819.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-libs/libgee/libgee-0.5.0.1.ebuild'
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-elementary/python-elementary-20090614.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-libs/libgee/libgee-0.5.0.1.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/python.eclass'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild'
!!!                 ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-9999.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '221' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-util/dialog/dialog-1.1.20080819.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-ecore/python-ecore-20090729.ebuild'
!!!   python-elementary-20090614.ebuild, line   7:  Called inherit 'enlightenment_svn-python'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.2.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.3.ebuild'
!!!                          ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/media-plugins/gst-plugins-ivorbis/gst-plugins-ivorbis-0.10.9.ebuild'
!!!          pyrtc-20090418.ebuild, line   5:  Called inherit 'distutils' 'git'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-9999.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/python.eclass'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-util/dialog/dialog-1.1.20080819.ebuild'
!!!                  python.eclass, line 277:  Called die
!!! The die message:
!!!  NEED_PYTHON variable is banned

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-ecore/python-ecore-20090729.ebuild'
!!!                         ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!!   xf86-video-glamo-9999.ebuild, line  10:  Called inherit 'x-modular'
!!!          libgee-0.5.0.1.ebuild, line   5:  Called inherit 'gnome2'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-ecore/python-ecore-20090729.ebuild'
!!!              pyrtc-9999.ebuild, line   5:  Called inherit 'distutils' 'git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-edje/python-edje-20090614.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.2.ebuild'
!!!   python-ecore-20090729.ebuild, line   7:  Called inherit 'enlightenment_svn-python'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                   ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 236' for eclass enlightenment_svn-python

received unknown com: export CATEGORY='dev-python' PR=r0 PVR=20090729 PV=20090729 EBUILD='/home/mgorny/repos/embedded-cross/dev-python/python-epsilon/python-epsilon-20090729.ebuild' P='python-epsilon-20090729' PF='python-epsilon-20090729' PN='python-epsilon'path
!!!     dialog-1.1.20080819.ebuild, line   5:  Called inherit 'eutils' 'autotools' 'toolchain-funcs'

!!! ERROR: dev-python/python-epsilon-20090614 failed.
!!!               gcc-4.4.3.ebuild, line  24:  Called inherit 'toolchain'
!!!                         ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'eutils'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'

!!!              ebuild-daemon.lib, line  37:  Called die
!!! ERROR: dev-python/python-etk-20090614 failed.
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 194' for eclass eutils

received unknown com: export CATEGORY='x11-libs' PR=r0 PVR='0.9.9.061' PV='0.9.9.061' EBUILD='/home/mgorny/repos/embedded-cross/x11-libs/ecore/ecore-0.9.9.061.ebuild' P='ecore-0.9.9.061' PF='ecore-0.9.9.061' PN=ecorepath
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 194' for eclass enlightenment_svn-python

received unknown com: export CATEGORY='x11-libs' PR=r0 PVR='0.9.9.062' PV='0.9.9.062' EBUILD='/home/mgorny/repos/embedded-cross/x11-libs/ecore/ecore-0.9.9.062.ebuild' P='ecore-0.9.9.062' PF='ecore-0.9.9.062' PN=ecorepath
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
Exception in thread Thread-15:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gnome2'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'toolchain'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-edje/python-edje-20090614.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.2.ebuild'
Exception in thread Thread-8:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!               ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'

!!! ERROR: dev-python/pygobject-2.18.0 failed.
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/media-plugins/gst-plugins-ivorbis/gst-plugins-ivorbis-0.10.9.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!               ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-edje/python-edje-20090614.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
!!!               ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '236' 'depend'
!!!               x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/pyxml/pyxml-0.8.4-r2.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!!               gcc-4.4.2.ebuild, line  24:  Called inherit 'toolchain'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!               ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   gst-plugins-ivorbis-0.10.9.ebuild, line   5:  Called inherit 'gst-plugins-bad'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   xf86-video-glamo-20100626.ebuild, line  16:  Called inherit 'x-modular'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '216' 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'

!!! ERROR: dev-python/python-etk-20090729 failed.
!!!               toolchain.eclass, line  29:  Called die
!!! The die message:
!!!  Need to upgrade to at least EAPI=2

!!!    python-edje-20090614.ebuild, line   7:  Called inherit 'enlightenment_svn-python'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'toolchain'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                  ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gst-plugins-bad'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 175' for eclass enlightenment_svn-python

received unknown com: export CATEGORY='x11-libs' PR=r0 PVR=20090729 PV=20090729 EBUILD='/home/mgorny/repos/embedded-cross/x11-libs/etk/etk-20090729.ebuild' P='etk-20090729' PF='etk-20090729' PN=etkpath
!!!                       ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-epsilon/python-epsilon-20090614.ebuild'
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-etk/python-etk-20090614.ebuild'
!!!                       ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-epsilon/python-epsilon-20090614.ebuild'
!!!               toolchain.eclass, line  29:  Called die
!!! The die message:
!!!  Need to upgrade to at least EAPI=2

!!!                  gnome2.eclass, line  19:  Called die
!!! The die message:
!!!  EAPI= is not supported

!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-etk/python-etk-20090614.ebuild'
!!!                   ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gst-plugins-bad.eclass'
!!!                   x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!           isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-epsilon/python-epsilon-20090614.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '203' 'depend'
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/gst-plugins-bad.eclass'
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   python-epsilon-20090614.ebuild, line   5:  Called inherit 'enlightenment_svn-python'
!!!                  ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: x11-libs/gtk+-2.14.5 failed.

!!! ERROR: x11-libs/ecore-0.9.9.063 failed.
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!     ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              gst-plugins-bad.eclass, line  19:  Called inherit 'eutils' 'gst-plugins10'
!!!                ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 165' for eclass enlightenment_svn-python

!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
received unknown com: export CATEGORY='x11-libs' PR=r0 PVR=9999 PV=9999 EBUILD='/home/mgorny/repos/embedded-cross/x11-libs/tslib/tslib-9999.ebuild' P='tslib-9999' PF='tslib-9999' PN=tslibpath
Exception in thread Thread-7:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '216' 'depend'
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: app-misc/mdbus-9999
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.1.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.2
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: dev-libs/libnl-2.0_pre9999
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: dev-python/pyrtc-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-elementary-20090614
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: dev-python/pyrtc-20090418
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-9999
!!! caught exception Metadata Exception: pkg ebuild src: sci-geosciences/fso-gpsd-0.8, attr eapi
error: parsed eapi doesn't match sourced eapi while processing ebuild src: sci-geosciences/fso-gpsd-0.8
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-efl-meta-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-ecore-20090729
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-20100626
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-edje-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/gst-python-0.10.16
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-epsilon-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-etk-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-epsilon-20090729
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/ecore-0.9.9.061
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/ecore-0.9.9.062
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-etk-20090729
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-devel/gcc-4.4.3
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/etk-20090729
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/freesmartphone-framework-20091021
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-devel/gcc-4.4.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-libs/libgee-0.5.0.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/ecore-0.9.9.063
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/tslib-9999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/fso-specs-2010.05.11.2
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild 
!!! ERROR: sys-mobilephone/fso-abyss-0.9.1 failed.
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-etk/python-etk-20090614.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gst-plugins10'

!!! ERROR: sys-mobilephone/freesmartphone-framework-20090704 failed.
!!!     python-etk-20090614.ebuild, line   5:  Called inherit 'enlightenment_svn-python'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/pygobject/pygobject-2.18.0.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 179' for eclass enlightenment_svn-python


!!! ERROR: sys-mobilephone/fso-alsa-1.0.0 failed.
received unknown com: export CATEGORY='dev-lang' PR=r0 PVR='0.8.1.1' PV='0.8.1.1' EBUILD='/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild' P='vala-0.8.1.1' PF='vala-0.8.1.1' PN=valapath
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/pygobject/pygobject-2.18.0.ebuild'
!!!                   ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gst-plugins10.eclass'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/pygobject/pygobject-2.18.0.ebuild'
!!!                         ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-etk/python-etk-20090729.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/gst-plugins10.eclass'
!!!                         ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!        pygobject-2.18.0.ebuild, line   5:  Called inherit 'alternatives' 'autotools' 'gnome2' 'python' 'virtualx'
!!!                gst-plugins10.eclass, line  37:  Called die
!!! The die message:
!!!  EAPI="0" is not supported anymore

!!!                         ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '296' 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '205' 'depend'
!!!                         ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '210' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '194' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gnome2'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-abyss/fso-abyss-0.9.1.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-etk/python-etk-20090729.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                                 ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-libs/ecore/ecore-0.9.9.063.ebuild'
 * 
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!!                  gnome2.eclass, line  19:  Called die
!!! The die message:
!!!  EAPI= is not supported

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'

!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5.5 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-etk/python-etk-20090729.ebuild'
!!! Call stack:

!!! ERROR: sys-mobilephone/libframeworkd-glib-0.1 failed.
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-libs/ecore/ecore-0.9.9.063.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-abyss/fso-abyss-0.9.1.ebuild'
!!!                                 ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'
!!!                      ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!          fso-alsa-1.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-abyss/fso-abyss-0.9.1.ebuild'
!!!              ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '176' 'depend'
!!!     python-etk-20090729.ebuild, line   5:  Called inherit 'enlightenment_svn-python'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-libs/ecore/ecore-0.9.9.063.ebuild'

!!! ERROR: sys-mobilephone/fsotdld-0.3.0 failed.
!!!         ecore-0.9.9.063.ebuild, line   8:  Called inherit 'toolchain-funcs' 'enlightenment' 'libtool'
!!!              ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: sys-mobilephone/fsoinitd-0.1.0 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'toolchain-funcs'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 200' for eclass toolchain-funcs

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='1.0.0' PV='1.0.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild' P='fso-raw-1.0.0' PF='fso-raw-1.0.0' PN='fso-raw'path
!!! die invoked from directory /usr/lib64/pkgcore
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 275' for eclass enlightenment_svn-python

!!! Call stack:
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.5' PV='0.5' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild' P='freesmartphone-framework-0.5' PF='freesmartphone-framework-0.5' PN='freesmartphone-framework'path
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!                      ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '245' 'depend'
!!!   freesmartphone-framework-20090704.ebuild, line   6:  Called inherit 'git'
!!!         fso-abyss-0.9.1.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'

!!! ERROR: sys-mobilephone/fsousaged-0.9.4 failed.
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!               ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!              ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                          ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                      ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '285' 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '203' 'depend'

!!! ERROR: sys-mobilephone/fsogsmd-0.5.0 failed.
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                      ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                      ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '198' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                              ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'

!!! ERROR: sys-mobilephone/fsodeviced-0.9.4 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!                      ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '208' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '198' 'depend'
!!!          isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!   libframeworkd-glib-0.1.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!          fsoinitd-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                              ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!    ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!           fsotdld-0.3.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!               ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'

!!! ERROR: sys-mobilephone/libframeworkd-glib-20090227 failed.
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 266' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090614 PV=20090614 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild' P='libframeworkd-glib-20090614' PF='libframeworkd-glib-20090614' PN='libframeworkd-glib'path
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
Exception in thread Thread-9:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                  isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '213' 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!   freesmartphone-framework-0.5.5.ebuild, line   6:  Called inherit 'git'
!!!            ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsousaged/fsousaged-0.9.4.ebuild'
!!!                       ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
!!!             gtk+-2.14.5.ebuild, line   7:  Called inherit 'gnome.org' 'flag-o-matic' 'eutils' 'libtool' 'virtualx'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsousaged/fsousaged-0.9.4.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'

!!! ERROR: sys-mobilephone/libfsotransport-0.9.6 failed.
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsousaged/fsousaged-0.9.4.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                   ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!         fsousaged-0.9.4.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'

!!! ERROR: sys-mobilephone/libframeworkd-glib-99999999 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'virtualx'
!!!           fsogsmd-0.5.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!                   ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                   ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '238' 'depend'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!        fsodeviced-0.9.4.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                   ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '266' 'depend'
!!!                virtualx.eclass, line  38:  Called die
!!! The die message:
!!!  virtualx eclass require EAPI=2 or newer.

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!                   ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsotransport/libfsotransport-0.9.6.ebuild'

!!! ERROR: sys-mobilephone/libphone-ui-shr-99999999 failed.
!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                   ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass cornucopia_git

!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-99999999.ebuild' P='libframeworkd-phonegui-99999999' PF='libframeworkd-phonegui-99999999' PN='libframeworkd-phonegui'path
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090724 PV=20090724 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090724.ebuild' P='libframeworkd-phonegui-20090724' PF='libframeworkd-phonegui-20090724' PN='libframeworkd-phonegui'path
src: sys-mobilephone/fso-abyss-0.9.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-20090704
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fso-alsa-1.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/vala-0.8.1.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/fsomusicd-0.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: media-plugins/gst-plugins-ivorbis-0.10.9
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/pygobject-2.18.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libframeworkd-glib-0.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsoinitd-0.1.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fsotdld-0.3.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fso-raw-1.0.0
ERROR pkgcore!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090506
: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fsousaged-0.9.4
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsogsmd-0.5.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsodeviced-0.9.4
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090227
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libfso-glib-2010.05.11.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090614
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libfsoframework-0.8.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/gsm0710muxd-0.9.3.1
!!! caught exception Metadata Exception: pkg ebuild src: sys-mobilephone/libfsotransport-0.9.6, attr eapi
error: parsed eapi doesn't match sourced eapi while processing ebuild src: sys-mobilephone/libfsotransport-0.9.6
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/gtk+-2.14.5
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libphone-ui-shr-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phon!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
Exception in thread Thread-12:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
Exception in thread Thread-11:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                           ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                           ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsotransport/libfsotransport-0.9.6.ebuild'
!!!                ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!               isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'

!!! ERROR: sys-mobilephone/libframeworkd-phonegui-20090614 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                   ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '266' 'depend'
!!!                ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '251' 'depend'
!!!                   ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsotransport/libfsotransport-0.9.6.ebuild'
!!!                       ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   libframeworkd-glib-20090227.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!                           ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
!!!   libfsotransport-0.9.6.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                           ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 255' for eclass git

!!!               isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='2010.05.11.2' PV='2010.05.11.2' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libfso-glib/libfso-glib-2010.05.11.2.ebuild' P='libfso-glib-2010.05.11.2' PF='libfso-glib-2010.05.11.2' PN='libfso-glib'failed
Exception in thread Thread-16:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   libframeworkd-glib-99999999.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!                       ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                        ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-ui-shr/libphone-ui-shr-99999999.ebuild'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 213' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='1.2.1' PV='1.2.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libgsm0710/libgsm0710-1.2.1.ebuild' P='libgsm0710-1.2.1' PF='libgsm0710-1.2.1' PN=libgsm0710path
!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 228' for eclass base

!!!                       ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '286' 'depend'
Exception in thread Thread-14:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.2' PV='0.9.2' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libgsm0710mux/libgsm0710mux-0.9.2.ebuild' P='libgsm0710mux-0.9.2' PF='libgsm0710mux-0.9.2' PN=libgsm0710muxpath
!!!                        ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-ui-shr/libphone-ui-shr-99999999.ebuild'
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!            isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-ui-shr/libphone-ui-shr-99999999.ebuild'
!!!                       ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   libphone-ui-shr-99999999.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!                               ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090614.ebuild'
!!!      ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                 ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: sys-mobilephone/libphone-ui-99999999 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                               ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090614.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '231' 'depend'
!!!                   isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090614.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   libframeworkd-phonegui-20090614.ebuild, line  21:  Called inherit 'shr'
!!!             ebuild-default-functions.lib, line 230:  Called __internal_inherit 'shr'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-ui/libphone-ui-99999999.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-ui/libphone-ui-99999999.ebuild'
!!!                        ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 193' for eclass shr

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-ui/libphone-ui-99999999.ebuild'
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='1.0.0' PV='1.0.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/mkdump/mkdump-1.0.0.ebuild' P='mkdump-1.0.0' PF='mkdump-1.0.0' PN=mkdumppath
Exception in thread Thread-13:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!    libphone-ui-99999999.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 224' for eclass base

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/ophonekitd/ophonekitd-99999999.ebuild' P='ophonekitd-99999999' PF='ophonekitd-99999999' PN=ophonekitdpath
Exception in thread Thread-5:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

egui-20090724
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libphone-utils-20090724
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libphone-utils-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfsobasics-0.9.9
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libgsm0710-1.2.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libgsm0710mux-0.9.2
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/ophonekitd-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libphone-ui-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/mkdump-1.0.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/phonefsod-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/ophonekitd-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/phoneuid-99999999
Error running an operation:
 Unhandled exception in regen_cache
 * Cache regen failed with 246
